tests/scripts/all.sh

```diff
@@ -897,26 +883,46 @@ component_test_full_cmake_clang () {
     if_build_succeeded env OPENSSL_CMD="$OPENSSL_NEXT" tests/compat.sh -e '^$' -f 'ARIA\|CHACHA'
 }
 
-component_build_deprecated () {
-    msg "build: make, full config + DEPRECATED_WARNING, gcc -O" # ~ 30s
+component_test_default_no_deprecated () {
+    msg "build: make, default + MBEDTLS_DEPRECATED_REMOVED" # ~ 30s
```

Comment:
Typo: the message says `REMOVED` but the code below has `WARNING`.

Reply:
Fixed in iteration 2. Note that I intend at least one more rebase because it turns out you can't just enable `MBEDTLS_PSA_CRYPTO_SE_C` in full since it uses GCC extensions (anonymous struct).